examples/loader-snowflake/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "type": "module",
+  "private": true,
+  "scripts": {
+    "clean": "rimraf src/.observablehq/cache",
+    "build": "rimraf dist && observable build",
+    "dev": "observable preview",
+    "deploy": "observable deploy",
+    "observable": "observable"
+  },
+  "dependencies": {
+    "@observablehq/framework": "^1.7.0",
+    "d3-dsv": "^3.0.1",
+    "d3-time": "^3.1.0",
+    "dotenv": "^16.4.5",
+    "snowflake-sdk": "~1.9.0"
```

Comment:
~1.9.0? is the ~ intentional? also, latest version of snowflake-idk is currently 1.10.1 released 3 weeks ago.

Reply:
Yes. The latest version of the snowflake-sdk generates a `snowflake.log` file by default and emits a bunch of cruft to stdout that breaks data loaders. Also the types in `@types/snowflake-sdk` are wrong because `snowflake-sdk` is distributed as CommonJS so `import * as snowflake from "snowflake-sdk"` doesn’t work.